### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'write' when packed with pyinstaller

### ISSUE THREAD

**Issue by nordicjg** [davidhozic/Tkinter-Async-Execute#12]
Hi, thanks for providing this package!
I get the following exception in my packed application (pyinstaller). If I run it as script everything works.

```
Traceback (most recent call last):
  File "tkinter\__init__.py", line 1967, in __call__
  File "tkinter\__init__.py", line 861, in callit
  File "tk_async_execute\widget.py", line 166, in write
AttributeError: 'NoneType' object has no attribute 'write'
```

**Comment by davidhozic** [comment]
Fixed in v1.3.2:
- https://pypi.org/project/tkinter-async-execute/
- ``pip install tkinter-async-execute==1.3.2``

**Comment by nordicjg** [comment]
Thank you very much for this quick fix! It works as expected now!
